sample-operators/mysql-schema/src/main/java/io/javaoperatorsdk/operator/sample/dependent/SchemaPollingResourceFetcher.java

```diff
@@ -18,7 +18,7 @@ public SchemaPollingResourceFetcher(MySQLDbConfig mySQLDbConfig) {
   }
 
   @Override
-  public Optional<Schema> fetchResource(MySQLSchema resource) {
-    return schemaService.getSchema(resource.getMetadata().getName());
+  public Set<Schema> fetchResources(MySQLSchema primaryResource) {
+    return null;
```

Comment:
Is this normal? If yes, can you add a comment explaining why, please?

Reply:
its not used delete.

Reply:
good catch! :)